src/frontend/js/utils/ObjectHelper/index.spec.ts

```diff
@@ -1,19 +1,27 @@
 import { ObjectHelper } from 'utils/ObjectHelper/index';
 
 describe('ObjectHelper', () => {
-  it('should return true if object is empty', () => {
-    expect(ObjectHelper.isEmpty({})).toBe(true);
-  });
+  describe('isEmpty', () => {
+    it('should return true if object is empty', () => {
+      expect(ObjectHelper.isEmpty({})).toBe(true);
+    });
 
-  it('should return false if object is not empty', () => {
-    expect(ObjectHelper.isEmpty({ foo: 'bar' })).toBe(false);
-  });
+    it('should return false if object is not empty', () => {
+      expect(ObjectHelper.isEmpty({ foo: 'bar' })).toBe(false);
+    });
+
+    it('should return true if array is empty', () => {
+      expect(ObjectHelper.isEmpty([])).toBe(true);
+    });
 
-  it('should return true if array is empty', () => {
-    expect(ObjectHelper.isEmpty([])).toBe(true);
+    it('should return true if array is not empty', () => {
+      expect(ObjectHelper.isEmpty(['foo'])).toBe(false);
+    });
   });
 
-  it('should return true if array is not empty', () => {
-    expect(ObjectHelper.isEmpty(['foo'])).toBe(false);
+  describe('omit', () => {
```

Comment:
We could maybe add a case which makes sure that there is no errors when trying to omit attributes not present? 

e.g. `ObjectHelper.omit({ foo: 1, bar: 2, baz: 3 }, 'foo', 'bazinga')`

Reply:
With the signature function, a typescript error is raised 😉

Reply:
Nice 😏